templates/etc/apache2/sites-available/apache__tpl_macros.j2

```diff
@@ -65,20 +176,7 @@ DocumentRoot {{ item.root|d(item.document_root) | quote }}
 {{ get_alias(item.alias, item.root|d(item.document_root)) }}
 {%- endif %}
 <Directory {{ item.root|d(item.document_root) | quote }}>
-    Options {{ debops__tpl_macros.get_yaml_list_for_elem(item.options|d(apache__vhost_options))|from_yaml | join(" ") }}
-    AllowOverride {{ debops__tpl_macros.get_yaml_list_for_elem(item.allow_override|d(apache__vhost_allow_override))|from_yaml | join(" ") }}
-
-{% if debops__tpl_macros.get_apache_version() | version_compare("2.4", ">=") %}
-    Require all granted
-{% else %}
-    Order allow,deny
-    Allow from all
-{% endif %}
-{% if item.root_directives|d() %}
-
-    {{ item.root_directives | indent(4) }}
-{% endif %}
-
+    {{ debops__tpl_macros.indent(get_document_root_directives(item), 4) -}}
 </Directory>
```

Comment:
Thanks for the review. I just double checked and it looks good :wink: If it makes Apache unhappy by failing to (re)start it that is easy to catch. Harder would be if it would not complain but do the wrong thing. Luckily, a syntax check already tells the problem in this case. But I can see the potential problem.

```
apache2ctl -t 
apache2: Syntax error on line 219 of /etc/apache2/apache2.conf: Syntax error on line 19 of /etc/apache2/sites-enabled/debops.apache-status.conf: Expected </Location> but saw </VirtualHost>
```